environments/staging/inventory.ini.j2

```diff
@@ -29,7 +29,7 @@ proxy
 web7
 web8
 
-{{ __pgproxy2__ }}
+{{ __pgproxy2__ }} pgbouncer_processes=2
```

Comment:
I think there is a command you need to run to make this appear in the actual inventory, that compiles the file from this template.

Reply:
Yes, that's:

`cchq <env> aws-fill-inventory [--cached]`

Reply:
Synced in last commit.